bin/ghe-host-check

```diff
@@ -36,7 +36,7 @@ set -e
 if [ $rc -ne 0 ]; then
     case $rc in
         255)
-            if echo "$output" | grep -i "port 22: connection refused\|port 22: no route to host\|ssh_exchange_identification: Connection closed by remote host\|Connection timed out during banner exchange" >/dev/null; then
+            if echo "$output" | grep -i "port 22: connection refused\|port 22: no route to host\|ssh_exchange_identification: Connection closed by remote host\|Connection timed out" >/dev/null; then
```

Comment:
To minimise the chances of this hiding issues with `122`, can we leave the `banner exchange` match as is, and add a new one for `port 22: Connection timed out`, so that it only matches on `22`?

Without this, it can get stuck in a loop if `22` and `122` both time out.